pkg/generic/thrift/parse.go

```diff
@@ -175,6 +180,9 @@ func addFunction(fn *parser.Function, tree *parser.Thrift, sDsc *descriptor.Serv
 		ID:   field.ID,
 		Type: reqType,
 	}
+	if opt != nil {
+		reqField.IsGoTagAliasDisabledFromOpt = opt.IsGoTagAliasDisabled
```

Comment:
`opt != nil` is an implicit condition. If a non-null opt is passed in, it will cause alias to be unexpectedly disabled. At this place, you should use WithDisableXXXOption. When setting this, mark a separate field to judge whether the user has set it.